scout/server/blueprints/institutes/views.py

```diff
@@ -89,3 +90,64 @@ def institute(institute_id):
     return render_template(
         "/overview/institute.html", form=form, default_phenotypes=default_phenotypes, **data
     )
+
+
+@blueprint.route("/<institute_id>/clinvar_submissions", methods=["GET", "POST"])
+@templated("overview/clinvar_submissions.html")
+def clinvar_submissions(institute_id):
+    """Handle clinVar submission objects and files"""
+
+    def generate_csv(header, lines):
+        yield header + "\n"
+        for line in lines:  # lines have already quoted fields
+            yield line + "\n"
+
+    if request.method == "POST":
+        submission_id = request.form.get("submission_id")
+        if request.form.get("update_submission"):
+            controllers.update_clinvar_submission_status(
+                store, request, institute_id, submission_id
+            )
+        elif request.form.get("delete_variant"):  # delete a variant from a submission
```

Comment:
Old code, but otherwise this would be a fairly straightforward case for flattening the control structure a bit. If the final else gets a simple condition..

Reply:
Simplified by restructuring the views